packages/utils/src/index.ts

```diff
@@ -0,0 +1,19 @@
+export * from './promise';
+export * from './array';
+export * from './file';
+export * from './object';
+export * from './random';
+export * from './string';
+export * from './validators';
+
+// Potential candidates for this package
```

Comment:
should we keep this here? do we have any other option where we can keep this? maybe create issue with TODO and link just issue here? no sure

Reply:
I totally agree with that, I created issue https://github.com/trezor/trezor-suite/issues/4786 for that and will remove it from here

Reply:
Nice, I think leaving just link to issue here will be enough